entrypoint.sh

```diff
@@ -2,7 +2,7 @@
 
 cd $BYTEWAX_WORKDIR
 . /venv/bin/activate
-python $BYTEWAX_PYTHON_FILE_PATH
+python -m bytewax.run $BYTEWAX_PYTHON_FILE_PATH
```

Comment:
So, we decided with esteban to keep this name here for compatibility with the helm chart. Also, since the first argument is required in the cli, just setting the env var wouldn't be enough, you still need to pass it as an argument, so the name of the env var itself is not relevant since it's not parsed by our cli. I should remove references to `$BYTEWAX_IMPORT_STR` though